src/stories/lib/make-story.js

```diff
@@ -1,5 +1,5 @@
 // @ts-ignore not worth helping TS understand this is valid since we don't need to type check this module
-// eslint-disable-next-line import/no-unresolved
+// eslint-disable-next-line
```

Comment:
I think it's an unwanted replacement, @Galimede it would be better to go for


```suggestion
// eslint-disable-next-line import-x/no-unresolved
```

Or maybe even customize the rule in our local project config to add `custom-elements.json`? I had not thought about that but it would be simpler since it would cover all cases 🤔